ui/apps/platform/src/Containers/Vulnerabilities/WorkloadCves/NamespaceView/NamespaceViewPage.tsx

```diff
@@ -161,11 +163,11 @@ function NamespaceViewPage() {
 
     return (
         <>
-            <PageTitle title="Workload CVEs - Namespace view" />
+            <PageTitle title={`${pageTitle} - Namespace view`} />
```

Comment:
Aha, we will need to double down on separation of subroute after information architecture is clearer?

Reply:
Likely yes. From what I can tell, the "Platform Components" filter _does_ work with the queries on the namespace route, so separate routes for both user and platform level workloads should be supported. I'll have to confirm with the team that this is what we want.